From 7328510f3b535531d04e5d6419e118f284aff599 Mon Sep 17 00:00:00 2001
From: Sergey Temerkhanov <s.temerkhanov@gmail.com>
Date: Wed, 4 Nov 2015 18:08:12 -0800
Subject: [PATCH 0062/1200] cavium: Update random ethernet address generation

---
 board/cavium/thunderx/thunderx.c | 22 +---------------------
 1 file changed, 1 insertion(+), 21 deletions(-)

diff --git a/board/cavium/thunderx/thunderx.c b/board/cavium/thunderx/thunderx.c
index 0450300a94..a3bd8ce512 100644
--- a/board/cavium/thunderx/thunderx.c
+++ b/board/cavium/thunderx/thunderx.c
@@ -137,26 +137,6 @@ int board_late_init(void)
 	return 0;
 }
 
-static void eth_random_enetaddr(uchar *enetaddr)
-{
-	uint32_t rval;
-	srand(get_timer(0));
-
-	rval = rand();
-	enetaddr[0] = rval & 0xff;
-	enetaddr[1] = (rval >> 8) & 0xff;
-	enetaddr[2] = (rval >> 16) & 0xff;
-
-	rval = rand();
-	enetaddr[3] = rval & 0xff;
-	enetaddr[4] = (rval >> 8) & 0xff;
-	enetaddr[5] = (rval >> 16) & 0xff;
-
-	/* make sure it's local and unicast */
-	enetaddr[0] = (enetaddr[0] | 0x02) & ~0x01;
-}
-
-
 /*
  * Board specific ethernet initialization routine.
  */
@@ -172,7 +152,7 @@ int board_eth_init(bd_t *bis)
 	unsigned char ethaddr[6];
 
 	if (!eth_getenv_enetaddr("ethaddr", ethaddr)) {
-		eth_random_enetaddr(ethaddr);
+		net_random_ethaddr(ethaddr);
 		eth_setenv_enetaddr("ethaddr", ethaddr);
 	}
 #endif
-- 
2.22.0

