From d30380c2bc3883507c1ccc71130dbbdd275ef783 Mon Sep 17 00:00:00 2001
From: Sergey Temerkhanov <s.temerkhanov@gmail.com>
Date: Fri, 6 Nov 2015 16:22:24 -0800
Subject: [PATCH 0080/1200] cavium: Suppress the build warning

---
 include/configs/thunderx_88xx.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/include/configs/thunderx_88xx.h b/include/configs/thunderx_88xx.h
index c670cd39e6..4139e9f29e 100644
--- a/include/configs/thunderx_88xx.h
+++ b/include/configs/thunderx_88xx.h
@@ -147,7 +147,6 @@
 #define CONFIG_CMD_DIAG
 #define CONFIG_CMD_ENV
 #define CONFIG_CMD_FLASH
-#define CONFIG_CMD_GO
 #define CONFIG_CMD_IMI
 #define CONFIG_CMD_MEMORY
 #define CONFIG_CMD_RUN
-- 
2.22.0

